unstructured/ingest/cli/cmds/chroma.py

```diff
@@ -0,0 +1,106 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import (
+    CliConfig,
+)
+from unstructured.ingest.connector.chroma import ChromaWriteConfig, SimpleChromaConfig
+
+
+@dataclass
+class ChromaCliConfig(SimpleChromaConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--path"],
+                required=False,
+                type=str,
+                help="Location where Chroma is persisted," "if not connecting via http.",
+            ),
+            click.Option(
+                ["--settings"],
+                required=False,
+                type=dict,
+                help="A dictionary of settings to communicate with the chroma server.",
+            ),
+            click.Option(
+                ["--tenant"],
+                required=False,
+                default="default_tenant",
+                type=str,
+                help="The tenant to use for this client.",
+            ),
+            click.Option(
+                ["--database"],
+                required=False,
+                default="default_database",
+                type=str,
+                help="The database to use for this client.",
+            ),
+            click.Option(
+                ["--host"],
+                required=False,
+                type=str,
+                help="The hostname of the Chroma server.",
+            ),
+            click.Option(
+                ["--port"],
+                required=False,
+                type=int,
+                help="The port of the Chroma server.",
+            ),
+            click.Option(
+                ["--ssl"],
+                required=False,
+                default=False,
```

Comment:
For bool types, it's nice to leverage the `is_flag` option
```suggestion
                default=False,
                is_flag=True,
```